### PR TITLE
Updates to JSONB datatype in YCQL

### PR DESCRIPTION
**Pull request by KavyaShivashankar**
1. Added an example to add new attributes via Update to JSONB datatype
2. Specified 'Secondary Indexes with JSONB datatype' as a YCQL feature
3. Corrected a typo in Data modelling comparison with Apache Cassandra

### CODE REVIEW

File: docs/content/preview/explore/json-support/jsonb-ycql.md [hunk 1]
```diff
@@ -38,7 +38,9 @@ JSON data types are for storing JSON (JavaScript Object Notation) data, as speci
 
 JSON functionality in YCQL is **a subset** of the [JSON functionality in PostgreSQL](https://www.postgresql.org/docs/11/datatype-json.html).
 
-YCQL supports the JSONB data type.
+YCQL supports the JSONB data type.  JSONB datatype stores JSON data in binary format. JSONB datatype does not preserve white space, does not preserve the order of object keys, and does not keep duplicate object keys. If duplicate keys are specified in the input, only the last value is kept.
```
Comment (ddhodge) [inline docs/content/preview/explore/json-support/jsonb-ycql.md:41]:
```suggestion
YCQL supports the JSONB data type. JSONB stores JSON data in binary format. JSONB does not preserve white space, does not preserve the order of object keys, and does not keep duplicate object keys. If duplicate keys are specified in the input, only the last value is kept.
```

File: docs/content/preview/explore/json-support/jsonb-ycql.md [hunk 2]
```diff
@@ -38,7 +38,9 @@ JSON data types are for storing JSON (JavaScript Object Notation) data, as speci
...
+
+The sections below walks you through different operations on JSONB column with some examples.
```
Comment (ddhodge) [inline docs/content/preview/explore/json-support/jsonb-ycql.md:43]:
```suggestion
The following sections describe different operations on JSONB columns with some examples.
```

File: docs/content/preview/explore/ycql-language/_index.md [hunk 3]
```diff
@@ -42,6 +42,7 @@ YCQL has a number of features that are not present in Cassandra, as summarized i
 | [Fast transactions](../../architecture/transactions/distributed-txns/) | Single round-trip distributed transactions |
 | [Native JSONB support](../../api/ycql/type_jsonb)| Enables document data modelling like MongoDB |
 | Fast and consistent Secondary Indexes | Immediately consistent indexes with point lookups (no fan-out) |
+| Secondary Indexes with JSONB datatype | Flexibity in data model without compromising Efficiency and Consistency |
```
Comment (ddhodge) [inline docs/content/preview/explore/ycql-language/_index.md:45]:
```suggestion
| Secondary indexes with JSONB datatype | Flexibility in data model without compromising efficiency and consistency |
```

**Review by ddhodge — APPROVED** [review]
minor comments

**Comment by premkumr** [comment]
LGTM

**Review by premkumr — APPROVED** [review]
LGTM

**Pull request merged**